src/OrbitQt/TrackTypeItemModelTest.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <QAbstractItemModelTester>
+#include <QModelIndex>
+#include <QSignalSpy>
+#include <QTableView>
+#include <QTest>
+#include <string>
+
+#include "QtUtils/AssertNoQtLogWarnings.h"
+#include "TimeGraphLayout.h"
+#include "TrackManager.h"
+#include "TrackManagerTest.h"
+#include "TrackTypeItemModel.h"
+
+namespace orbit_qt {
+
+class TrackTypeItemModelTest : public ::testing::Test {
+ public:
+  TrackTypeItemModelTest()
+      : Test(),
+        capture_data_(orbit_gl::TrackTestData::GenerateTestCaptureData()),
+        track_manager_(nullptr, nullptr, &layout_, nullptr, capture_data_.get()) {}
+
+  int FindRowByTrackType(Track::Type track_type) {
+    int found_row = -1;
+
+    for (int i = 0; i < model_.rowCount(); ++i) {
+      Track::Type this_row_type =
+          model_.data(model_.index(i, 0), Qt::UserRole).value<Track::Type>();
+      if (track_type == this_row_type) {
+        found_row = i;
+      }
+    }
+
+    EXPECT_TRUE(found_row != -1);
```

Comment:
Can the test continue if this fails? If not, it should probably be an `ASSERT` in the test body.

Reply:
Yeah, it shouldn't be here, I moved this to the tests that actually expect this condition.